.github/workflows/dockerbuild.yml

```diff
@@ -1,13 +1,9 @@
 name: Build and Publish Docker Images
 
 on:
-# workflow_run:
-#   workflows:
-#     - "Lint and Test Python Package"
-#   branches:
-#     - main
-#   types:
-#     - completed
+  push:
+    branches: [ master ]
+    tags: [ 'v*.*.*' ]
```

Comment:
These rules should be okay for now.

However, I would prefer to have some kind of rule in place to ensure that this workflow can only be triggered after the `package-lint-and-test` workflow was successful. But that can still be added in the future.